src/main/resources/answers.properties

```diff
@@ -27,6 +27,8 @@ answers.start.description=Just start command
 answers.start.message=Hi, i'm a Blamer!
 answers.token.command=/token
 answers.token.description=Set GitHub token to get updates
-answers.token.message=You need to provide me GitHub token with notifications access! Send it with /registry your_token
+answers.token.message=You need to provide me GitHub token with notification access! Send it with /registry your_token
 answers.registry.command=/registry
 answers.registry.description=Token authentication
+answers.chat-id.command=/chatid
+answers.chat-id.description=Saves your chat id
```

Comment:
@l3r8yJ maybe it will be better to say not chat id but something like 'Saves your telegram chat'

Most telegram users don't know what is chat id

WDYT?

Reply:
@h1alexbel sure